application/features/mfa.feature

```diff
@@ -62,6 +62,43 @@ Feature: MFA
         | label     | A Label        |
         | verified  | 1              |
 
+  Scenario: Create new MFA record of type recovery
+    Given I provide the following valid data:
+      | property       | value                |
+      | employee_id    | 123                  |
+      | type           | recovery             |
+      | label          | A Label              |
+      | recovery_email | recovery@example.com |
+    When I request "/mfa" be created
+    Then the response status code should be 200
+    And the response should contain a data array with 0 items
+    And an MFA record exists for an employee_id of "123"
+    And the following MFA data should be stored:
+      | property       | value                |
+      | type           | recovery             |
+      | label          | A Label              |
+      | verified       | 1                    |
+      | recovery_email | recovery@example.com |
+
+  Scenario: Create new MFA record of type recovery with no recovery email
+    Given I provide the following valid data:
+      | property    | value    |
+      | employee_id | 123      |
+      | type        | recovery |
+      | label       | A Label  |
+    When I request "/mfa" be created
+    Then the response status code should be 400
+
+  Scenario: Create new MFA record of type recovery with an invalid recovery email
+    Given I provide the following valid data:
+      | property       | value        |
+      | employee_id    | 123          |
+      | type           | recovery     |
+      | label          | A Label      |
+      | recovery_email | invalidEmail |
```

Comment:
The wording of this test seems contradictory: "the following valid data... invalidEmail".

We might have a wording "bug" in our behat tests for ID Broker. However, it looks like other tests already have this problem, so this is likely something to fix in a later PR. If you would, please make a note (possibly in this same YouTrack issue) to come back and look at this later.